howto/getting-started/deploy-workload-manager.md

```diff
@@ -0,0 +1,163 @@
+---
+relatedlinks: "[Get&#32;started&#32;with&#32;LXD](https://documentation.ubuntu.com/lxd/en/latest/tutorial/first_steps/), [Get&#32;started&#32;with&#32;Juju](https://juju.is/docs/juju/tutorial), [Slurm&#32;website](https://slurm.schedmd.com/overview.html)"
+---
+
+# Deploy the workload manager
+
+An HPC workload manager is the base component of your Charmed HPC cluster.
+All the applications and auxiliary services that can be deployed with Charmed HPC
+are integrated with the workload manager to compose the functionality of your cluster.
```

Comment:
I think it's not common to have explanations like this on how-to's, they're more dry on their delivery so that it's easier to find the important information.

Reply:
I can abridge this:

> This guide shows you how to deploy the workload manager of your Charmed HPC cluster